packages/office-ui-fabric-react/src/components/Panel/Panel.types.ts

```diff
@@ -168,11 +168,6 @@ export interface IPanelProps extends React.Props<Panel> {
    * Custom renderer for content in the sticky footer
    */
   onRenderFooterContent?: IRenderFunction<IPanelProps>;
-
-  /**
-   * Internal ID passed to render functions.
-   */
-  componentId?: string;
```

Comment:
You can't remove this; it will break consumers. Please Add a `@deprecation` tag to the comments and leave it intact so that we don't cause build breaks.

I'm not sure this is something we need to keep respecting though, because it doesn't change any behavior of the component. But, it would be good to add a `warnDeprecations` call in the constuctor of panel to indicate to partners to remove it.

Reply:
Done.